packages/fx-core/src/component/driver/aad/create.ts

```diff
@@ -90,7 +90,8 @@ export class CreateAadAppDriver implements StepDriver {
           )
         );
         // Create new AAD app if no client id exists
-        const aadApp = await aadAppClient.createAadApp(args.name);
+        const signInAudience = args.signInAudience ?? "AzureADMyOrg";
```

Comment:
how about have a variable for audience types to wrap the different types "AzureADMyOrg".....

Reply:
And SignInAudience should be enum right?

Reply:
Sure I will update here. One potential problem is that we need to update our code if AAD updates its schema.

Reply:
Updated, thx!